arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,425 @@
+
+| Error                   | Description                                                                                                 |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `MALFORMED_DATA`        | wrong input data, failed to deserialize, missing certain fields, etc.                                       |
+| `NOT_DEPOSITOR`         | the address is not an active depositor in the auction contract                                              |
+| `WRONG_CHAIN_ID`        | wrong chain id for the target chain                                                                         |
+| `WRONG_SIGNATURE`       | signature failed to verify                                                                                  |
+| `BAD_ROUND_NUMBER`      | incorrect round, such as one from the past                                                                  |
+| `RESERVE_PRICE_NOT_MET` | bid amount does not meet the minimum required reserve price on-chain                                        |
+| `INSUFFICIENT_BALANCE`  | the bid amount specified in the request is higher than the deposit balance of the depositor in the contract |
+
+### Step 3: find out the winner of the auction
+
+After the auction closes, and before the round starts, the autonomous auctioneer will call the auction contract with the two highest bids received, so the contract can declare the winner and subtract the second-highest bid from the winner's deposited funds. After this, the contract will emit an event with the new express lane controller address.
+
+We can use this event to determine whether or not we've won the auction. The event signature is:
+
+```solidity
+event SetExpressLaneController(
+    uint64 round,
+    address indexed previousExpressLaneController,
+    address indexed newExpressLaneController,
+    address indexed transferor,
+    uint64 startTimestamp,
+    uint64 endTimestamp
+);
+```
+
+Here's an example to get the log from the auction contract to determine the new express lane controller:
+
+```tsx
+const fromBlock = <any recent block, for example during the auction>
+const logs = await publicClient.getLogs({
+  address: auctionContractAddress,
+  event: auctionContractAbi.filter((abiEntry) => abiEntry.name === 'SetExpressLaneController')[0],
+  fromBlock,
+});
+
+const newExpressLaneController = logs[0].args.newExpressLaneController;
+console.log(`New express lane controller: ${newExpressLaneController}`);
+```
+
+If you won the auction, congratulations! You are the express lane controller for the next round, which, by default, will start 15 seconds after the auction closed. The following section explains how we can submit a transaction to the express lane.
```

Comment:
Oh, I didn't know that. Why does it close at 00:49 though? We are saying in other parts of the docs that it closes 15 seconds before the new round starts (and there's the `auctionClosingSeconds` parameter that defaults to 15 iirc.

Reply:
Ah okay i think i am wrong here: It closes at 00:00:45 but the auction may be resolved at 00:00:49, for example. Resolving this comment because your original text is correct